### PR TITLE
Passengers counter

### PR DESCRIPTION
**Pull request by oriefrati**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/Components/SelectModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import { Text, View } from 'react-native';
+import styled from 'styled-components';
+import SelectDropdown from 'react-native-select-dropdown';
```
Comment (OfryL) [inline examples/client/Locomotion/src/Components/SelectModal/index.tsx:4]: move to another file for only importing this

**Review by OfryL — APPROVED** [review]

File: examples/client/Locomotion/src/Components/SelectModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import SvgIcon from '../SvgIcon';
+import person from '../../assets/person.svg';
+import {
+  FONT_SIZES, FONT_SIZES_VALUES, FONT_WEIGHTS,
+} from '../../context/theme';
+
+const hextToRgba = (hex: string, alpha: number) => {
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/Components/SelectModal/index.tsx]: move to utils

File: examples/client/Locomotion/src/Components/SelectModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  <StyledRow selected={selected}>
+    <StyledIcon
+      Svg={person}
+      selected={selected}
+    />
+    <StyledText selected={selected}>{item.label}</StyledText>
+
+  </StyledRow>
+);
+const SelectModal = ({ data, onSelect, onError }) => {
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/Components/SelectModal/index.tsx]: add props interface

File: examples/client/Locomotion/src/Components/SelectModal/index.tsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  const [selectedItem, setSelectedItem] = useState(null);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    if (data && data.length) {
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/Components/SelectModal/index.tsx]: data?.length

File: examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/RideButtons/index.tsx [hunk 5]
```diff
@@ -219,6 +230,13 @@ const RideButtons = ({
     );
   };
 
+  useEffect(() => {
+    if (!chosenService || (chosenService.pooling && chosenService.pooling === 'no')) {
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/RideButtons/index.tsx]: use/create pooling consts

File: examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/RideButtons/index.tsx [hunk 6]
```diff
@@ -233,17 +251,27 @@ const RideButtons = ({
...
-          changeBsPage(BS_PAGES.CONFIRM_PICKUP);
-        }}
-      >
-        <ButtonText testID="select">
-          {i18n.t('general.select').toString()}
-        </ButtonText>
-      </StyledButton>
+      <RowContainer>
+
+        {chosenService && chosenService.pooling && chosenService.pooling !== 'no'
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/RideButtons/index.tsx]: chosenService && chosenService.pooling -> chosenService?.pooling

**Pull request merged**